Gamma_Analysis.py

```diff
@@ -16,7 +16,10 @@
 import os
 
 EFFICIENCY_CAL_COEFFS = [-5.1164, 161.65, -3952.3, 30908]
-
+isotope_list = [ii.potassium_40, ii.bismuth_214, ii.thallium_208,
+                ii.caesium_137, ii.caesium_134, ii.cobalt_60,
+                ii.actinium_228, ii.lead_212, ii.lead_214,
+                ii.thorium_234, ii.lead_210]
```

Comment:
Chris removed extra isotopes a few days ago. It saves time for saving peak images to file. So `isotope_list` should just be K-40, Bi-214, Tl-208, Cs-137, Cs-134.